aips/aip-119.md

```diff
@@ -0,0 +1,230 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Scheduled Transactions
+author: Manu Dhundi (https://github.com/manudhundi) & Zekun Li (https://github.com/zekun000)
+discussions-to (*optional):
+Status: Draft
+last-call-end-date (*optional):
+type: Core, Framework
+created: 04/15/2025
+updated (*optional):
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-119 - Scheduled Transactions
+  
+(Please give a temporary file name to your AIP when first drafting it, such as `aip-x.md`. The AIP manager will assign a number to it after reviewing.)
+
+(Please remove the questions in the "quote box". Provide complete context to the questions being asked in the content that you provide.)
+
+## Summary
+A scheduled transaction is a user-submitted transaction that specifies another transaction to be executed at a future time. The system executes it in a suitable future block.  
+ 
+This AIP proposes framework changes to support scheduling, gas handling, and deposits for scheduling future transactions. It also introduces core changes for the blockchain to check at every block for ready scheduled transactions, prioritize them alongside others based on gas fees and existing rules, and execute them accordingly.
+
+### Out of scope
+ * Privacy
```

Comment:
nit: it might be worth elaborating on each bullet point (even if only briefly), e.g.:

```
* Privacy: Scheduled transactions will be stored on-chain. Providing privacy for them is out of scope.
* Additional triggers: Other triggers (e.g., event-based triggers) are not targeted by this AIP.
...
```